mast/testRunner.mt

```diff
@@ -163,6 +163,37 @@ def makeAsserter() as DeepFrozen:
                     catch _:
                         successes += 1
 
+                to contains(o, k):
+                    if (o._respondsTo("contains", 1)):
```

Comment:
why bother with the `o._respondsTo("contains", 1)` checks?

If `o.contains(k)` throws `refused`, that's a bug that should be reported as such, no? Why change it to `assert.fail()`?